### PR TITLE
Added Mark Headd to list of authors, team members

### PR DESCRIPTION
**Pull request by gemfarmer**
[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/mark-headd-author)

[:sunglasses: PREVIEW](https://federalist.18f.gov/preview/18F/18f.gsa.gov/mark-headd-author/)

Changes proposed in this pull request:
- added mark to authors.yml
- added mark to _team directory

/cc @gboone @mheadd

### CODE REVIEW

**Comment by gemfarmer** [comment]
Mark submitted the original PR to `staging` #1973 by following this link:
https://pages.18f.gov/blogging-guide/metadata/

The documentation for this blog guide needs to be altered to avoid further PRs to staging

**Comment by coreycaitlin** [comment]
@gemfarmer I think I made the change we need in the blogging guide — let me know if I missed something!

**Comment by gemfarmer** [comment]
@coreycaitlin It looks like it is still linking to staging, not dev. Are these changes live or do I need to view them somewhere else?

**Comment by coreycaitlin** [comment]
@gemfarmer that change is now merged in the blogging guide, thanks to @awfrancisco 🎉

File: _team/mheadd.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+first_name: Mark
+last_name: Headd
+full_name: Mark Headd
+role: Innovation Specialist
+city: Syracuse
+state: New York
+github: mheadd
+twitter: mheadd
+team: Acquisitions
+---
```
Comment (gboone) [inline _team/mheadd.md]: Should this file be in `_authors/` instead of `_team/`?

**Comment by gboone** [comment]
I'm sorta out of ideas on why the tests are failing on this one. Looks like maybe Phantom is running long and the failing out? Maybe a Circle configuration we need to check out? I don't know why it'd only happen on this branch and not others, though.

**Comment by mugizico** [comment]
@gboone it is actually happening on other PR-ready branches as well.

**Pull request merged**